src/main/java/org/jabref/logic/importer/ParserResult.java

```diff
@@ -35,7 +34,7 @@ public ParserResult() {
     }
 
     public ParserResult(Collection<BibEntry> entries) {
-        this(BibDatabases.createDatabase(BibDatabases.purgeEmptyEntries(entries)));
+        this(new BibDatabase(BibDatabases.purgeEmptyEntries(entries)));
```

Comment:
This is a kind of taste, isn't it? - Would assume that the Builder pattern would be preferred.

Reply:
Builder pattern for a constructor with one argument seems a bit overkill, don't you think?